### PR TITLE
[ADD] seeding the db with random info

### PR DESCRIPTION
**Pull request by RainproDash**
Added some random information into the db by using factories and faker

### CODE REVIEW

**Review by jclge — CHANGES_REQUESTED** [review]
Resolve conflicts 👁️👄👁️

File: backend/src/seeder/seed.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    username: PGUSER!,
+    password: PGPASSWORD!,
+    database: PGDATABASE!,
+    entities: [Inventory, Project, User],
+    logging: true,
+    synchronize: true,
+    ssl: true,
+    factories: [UserFactory, InventoryFactory, ProjectsFactory],
+    seeds: [MainSeeder],
+};
```
Comment (thmarinho) [inline backend/src/seeder/seed.ts]: Factorise this with db.ts file

File: backend/src/seeder/seed.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+const dataSource = new DataSource(options);
+
+dataSource.initialize().then(async () => {
+    await dataSource.synchronize(true);
+    await runSeeders(dataSource);
+    process.exit();
```
Comment (thmarinho) [inline backend/src/seeder/seed.ts]: Useless line

File: backend/package.json [hunk 3]
```diff
@@ -9,7 +9,8 @@
     "build": "tsc",
     "test": "npm run build && DEBUG=true node --experimental-vm-modules node_modules/jest/bin/jest.js -i",
     "lint": "eslint . --ext .ts",
-    "lint:fix": "eslint . --ext .ts --fix"
+    "lint:fix": "eslint . --ext .ts --fix",
+    "seed" : "tsc ./src/seeder/seed.ts"
```
Comment (thmarinho) [inline backend/package.json]: How does compiling run the seeder ?

File: backend/src/seeder/seed.ts [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    logging: true,
+    synchronize: true,
+    ssl: true,
+    factories: [UserFactory, InventoryFactory, ProjectsFactory],
+    seeds: [MainSeeder],
+};
+
+const dataSource = new DataSource(options);
+
+dataSource.initialize().then(async () => {
```
Comment (thmarinho) [inline backend/src/seeder/seed.ts]: Add a condition to make sure this won't be executed in prod

**Review by thmarinho — CHANGES_REQUESTED** [review]

File: backend/src/seeder/seed.ts (hunk 2 above)
Comment (RainproDash) [reply backend/src/seeder/seed.ts]: The process.exit is used to exit the program after the seeding has run. If it doesn't exist we have to exit the program from the terminal

**Review by thmarinho — APPROVED** [review]

**Pull request merged**